src/reducers/passwordReducer.js

```diff
@@ -0,0 +1,27 @@
+import { PASSWORD_FAIL, PASSWORD_SUCCESS } from './types';
+
+const initialState = {
+  emailSuccess: false,
```

Comment:
include email in initial state if you're gonna return it in reducer, otherwise whatever you attach to the email state should bug out before password success is called